server/repository/src/db_diesel/stock_line.rs

```diff
@@ -160,15 +161,27 @@ impl<'a> StockLineRepository<'a> {
 type BoxedStockLineQuery = IntoBoxed<
     'static,
     LeftJoin<
-        LeftJoin<LeftJoin<InnerJoin<stock_line::table, item::table>, location::table>, name::table>,
+        LeftJoin<
+            LeftJoin<
+                InnerJoin<stock_line::table, InnerJoin<item_link::table, item::table>>,
+                location::table,
+            >,
+            name::table,
+        >,
         barcode::table,
     >,
     DBType,
 >;
 
+allow_tables_to_appear_in_same_query!(stock_line, item_link);
```

Comment:
Can these be moved to their respective row files?

Reply:
Sounds good!

Reply:
Done here 5f9ba3fe376ce8d81d436265440d28ab37c3be32